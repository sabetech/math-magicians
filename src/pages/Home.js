import React from 'react';

function Home() {
  return (
    <div>
      <h2>Welcome to our page</h2>
      <p className="paragraph">
        Welcome to our calculator application! We are here to assist you in all your mathematical needs. 
    Whether you&apos;re looking to perform simple arithmetic or more complex calculations, our easy-to-use 
    interface is here to help. Please feel free to ask us any questions you may have, and we&apos;ll do our best 
    to assist you. Let&apos;s get started!
      </p>
      <p className="paragraph">
        Use the links above to navigate to the various pages!
      </p>
    </div>
  );
}

export default Home;
