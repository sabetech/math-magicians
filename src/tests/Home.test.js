import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/Home';

describe('Calculator pages that should render correctly', () => {
  it('Home page should render correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
