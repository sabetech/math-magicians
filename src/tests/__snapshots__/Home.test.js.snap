// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator pages that should render correctly Home page should render correctly 1`] = `
<div>
  <h2>
    Welcome to our page
  </h2>
  <p
    className="paragraph"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc id suscipit est. Duis at tortor sed mauris porta finibus. Nam ac mauris sed est scelerisque placerat non ut elit. Nulla facilisi. Vivamus egestas libero vel libero ornare, eget aliquet dolor convallis. Donec efficitur metus nec erat pretium maximus. Suspendisse efficitur nibh eget purus lacinia, id aliquet felis auctor. Cras nec cursus eros. Donec erat lectus, venenatis quis nibh eget, congue commodo quam. Ut accumsan dictum quam a luctus. Nullam pellentesque ante eros, ut pharetra leo finibus sed. Aliquam augue nisi, volutpat ac posuere placerat, condimentum eget dui. Donec convallis, ipsum eu viverra tempus, quam diam elementum purus, vel eleifend nibh nunc at est. Nunc vestibulum sem enim, at ultrices lacus sodales eget. Curabitur a nisi dolor. Vestibulum convallis nec dui id sagittis.
  </p>
  <p
    className="paragraph"
  >
    Morbi quis eleifend elit, in ornare neque. Ut tincidunt efficitur euismod. Donec eget convallis dui, sed sollicitudin ex. Interdum et malesuada fames ac ante ipsum primis in faucibus. Curabitur vestibulum aliquet placerat. Phasellus molestie tempus aliquet. Vivamus venenatis tristique eleifend.
  </p>
</div>
`;
